gslab_scons/_release_tools.py

```diff
@@ -238,6 +238,70 @@ def up_to_date(mode = 'scons', directory = '.'):
     return bool(result)
 
 
+def list_ignored_files(dir_path = './release'):
+    '''List files ignored by git 
+
+    This function returns a list of the files in the directory
+    given by the dir_path argument that are ignored by git. 
+    '''
+    message = subprocess.check_output('git status --ignored', shell = True)
+    message = message.split('\n')
+    message = map(lambda s: s.strip(), message)
+
+    try:
+        ignored_line = message.index('Ignored files:')
+    except ValueError:
+        # If the git status message lists no ignored files, return empty list
+        return []
+
+    # Use normpath() so paths can be compared as strings
+    dir_path = os.path.normpath(dir_path)
+    ignore_dirs  = []
+    ignore_files = []
+    for line in message[ignored_line:len(message)]:
+        # Determine if line could be in path's directory
```

Comment:
I don't quite follow this comment. The phrase "path's directory" is unclear to me.